jetstream/workflows/run.yaml

```diff
@@ -26,6 +25,15 @@ spec:
           - name: date
             value: "{{item.date}}"
         withParam: "{{inputs.parameters.experiments}}"  # process these experiments in parallel
+        continueOn:
+          failed: true
```

Comment:
This is necessary, otherwise if one of the analysis steps fails it will fail the entire workflow